server/src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -44,6 +44,9 @@ object CromwellEntryPoint extends GracefulStopSupport {
 
   val gracefulShutdown = config.as[Boolean]("system.graceful-server-shutdown")
 
+  // 3 minute DNS TTL down from JVM default of infinite [BA-6454]
+  java.security.Security.setProperty("networkaddress.cache.ttl" , "180")
```

Comment:
How I chose
![Screen Shot 2020-06-16 at 10 44 57 AM](https://user-images.githubusercontent.com/1087943/84789651-792f2000-afbe-11ea-9d2f-d8e67c939d07.png)

Reply:
60 seconds for a DNS lookup sounds absolutely glacial to me, but I bow to Rob's far greater knowledge on these things.

Reply:
Glacial? That is among the fastest commonly used I believe. (This is the time to cache, not the request time)

Reply:
Oh! Cache lifetime - that makes way more sense and is totally reasonable.